server/src/main/java/io/crate/metadata/view/ViewInfoFactory.java

```diff
@@ -62,9 +62,12 @@ public ViewInfo create(RelationName ident, ClusterState state) {
         }
         List<Reference> columns;
         boolean analyzeError = false;
+        boolean errorOnUnknownObjectKey = view.errorOnUnknownObjectKey();
         try {
             CoordinatorTxnCtx transactionContext = CoordinatorTxnCtx.systemTransactionContext();
             transactionContext.sessionSettings().setSearchPath(view.searchPath());
+            transactionContext.sessionSettings().setErrorOnUnknownObjectKey(errorOnUnknownObjectKey);
```

Comment:
This is the place where the setting is now applied to the analyzer part of the query from the view.